app/src/pages/App.js

```diff
@@ -1,22 +1,38 @@
-import { Header } from './header';
+import { Header, MovieHeader } from './header';
 import { Movies } from './movies';
 import { Footer } from './footer';
+import { Page404 } from './404';
 import { ErrorBoundary } from '@globalComponents';
 import { Provider } from 'react-redux';
-import { useState } from 'react';
 import { store } from '@store/store';
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
 
 function App() {
-    const [movieId, setMovieId] = useState(null);
-
     return (
-        <ErrorBoundary>
-            <Provider store={store}>
-                <Header showSearchPanel={setMovieId} movieId={movieId} />
-                <Movies showMovieDetails={setMovieId} />
-                <Footer />
-            </Provider>
-        </ErrorBoundary>
+        <Router>
+            <ErrorBoundary>
+                <Provider store={store}>
+                    <Switch>
+                        <Route path='/film/:movieId'>

```

Comment:
accumulate in one file all navigationMap

Reply:
Done